static/app/components/acceptanceTestTooltip.tsx

```diff
@@ -0,0 +1,20 @@
+import {useEffect, useState} from 'react';
+
+import {DO_NOT_USE_TOOLTIP, TooltipProps} from 'sentry/components/tooltip';
+import {tooltipStore} from 'sentry/stores/tooltipStore';
+import domId from 'sentry/utils/domId';
+
+export function AcceptanceTestTooltip(props: TooltipProps) {
+  const [open, setOpen] = useState<undefined | boolean>(undefined);
```

Comment:
shouldn't this be always a boolean? instead of `undefined`, we could use `false`

Reply:
_Should_  we make it always a boolean?
and instead have another state explicitly for 'in control' or not?

would be a bit odd if we ever check for is true, is false, is undefined

Reply:
Tristate bools are usually a 'gotcha', might be better in the long term to split into 2 bools, a controlled and controlled state.

Reply:
100%, that would also be useful if we wanted to reset controlled state. The tooltip component has that optional forceShow open, so here the undefined state kinda reflects that (altho poorly rn)